### PR TITLE
EN-2265 Fixed DE component lastUpdate date format and other little improvements

### PR DESCRIPTION
**Pull request by zonia3000**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/aps/system/services/digitalexchange/model/ResilientPagedMetadata.java [hunk 1]
```diff
@@ -30,10 +30,12 @@
 
     public ResilientPagedMetadata() {
         super();
+        this.errors = new ArrayList<>();
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/aps/system/services/digitalexchange/model/ResilientPagedMetadata.java]: Any reason not initialize this on the member variable?
```
@JsonIgnore
private List<RestError> errors = new ArrayList<>();
```

**Comment by zonia3000** [comment]
Hi @joewhite101, I added the "Work in Progress" label because Giuseppe reported me an issue with components filtering. I'll fix it and I'll commit changes on this branch since it is related to EN-2265 task

File: engine/src/main/java/org/entando/entando/aps/system/services/digitalexchange/model/ResilientPagedMetadata.java (hunk 1 above)
Comment (zonia3000) [reply engine/src/main/java/org/entando/entando/aps/system/services/digitalexchange/model/ResilientPagedMetadata.java]: No, I moved it there.

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by zonia3000** [comment]
@joewhite101 As pointed out by Eugenio, the default operator should be "eq" for each filter, however inside [Filter class](https://github.com/entando/entando-core/blob/master/engine/src/main/java/org/entando/entando/web/common/model/Filter.java) the default operator is "like". I tried to change like into eq but tests failed. I think that we should try to align the default operator in the various parts of the code, but probably this requires a dedicated ticket.

**Comment by entando-service** [comment]
All is well

**Pull request merged**